### ISSUE TITLE
Different results compare to Matlab's fft

### ISSUE THREAD

**Issue by xdelen** [dntj/jsfft#7]
Hey,
Comparing the results with what you can obtain with matlab, I noticed some artifacts in the fft output. For example, it is the case for a 65536 points signal with a rectangular signal of 1/10th of the total width. Some "butterfly" shaped artifacts are visible at high frequencies. Do you have an idea where it is coming from?
Thanks,
Xavier

**Comment by dntj** [comment]
Hmmm, weird, can you provide any more info?  A screenshot or actual data?

**Comment by xdelen** [comment]
Hey, 

Thank you for the answer. I simply took the example, changed the number of points for 65536 and line 100 to define a square signal with a width of 1/10th of the total window width using this line: value.real = (i > n/2-n/20 && i < n/2+n/20) ? 1 : 0

Trying to reproduce what I have seen on Friday I noticed that it actually depends on the web browser which I use. The screenshots below show the results for Chrome and Firefox. Chrome works fine but Firefox doesn't. Any clue where this could be coming from?

![fft_test](https://cloud.githubusercontent.com/assets/21105108/18826558/640bae8e-83ce-11e6-8e93-4db21b1df8f2.png)
